src/extension.ts

```diff
@@ -26,7 +28,20 @@ export function activate(context: vscode.ExtensionContext) {
         nisabaOutputChannel.append(str + '\n');
     });
 
-    context.subscriptions.push(disposable);
+    const disposable2 = vscode.commands.registerCommand('ucl-rsdg.validationButton', () => {
+        // The code you place here will be executed every time your command is 
+        const firstLine = vscode.window.activeTextEditor.document.lineAt(0);
+        const lastLine = vscode.window.activeTextEditor.document.lineAt(vscode.window.activeTextEditor.document.lineCount - 1);
+        const textRange = new vscode.Range(firstLine.range.start, lastLine.range.end);
+        const filePath = vscode.window.activeTextEditor.document.uri.fsPath;
+        const fileProject = "cams/gkab";
+        const fileContent = vscode.window.activeTextEditor.document.getText(textRange);
+        // The validate function is currently not mapped to the appropriate logging functions
+        validate(filePath,fileProject,fileContent)
+        });
+
+    context.subscriptions.push(disposable1);
```

Comment:
Minor point, but I would keep this line before we define `disposable2`, so that it's closer to the values it concerns (`disposable1`). Otherwise it's easier to change one use of `disposable1` and forget the other.